torch/_torch_docs.py

```diff
@@ -5719,14 +5719,23 @@
 
 add_docstr(torch.zeros,
            r"""
-zeros(*sizes, out=None) -> Tensor
+zeros(size, out=None, dtype=None, layout=torch.strided, device=None, requires_grad=False) -> Tensor
 
 Returns a tensor filled with the scalar value `0`, with the shape defined
-by the variable argument :attr:`sizes`.
+by the tuple :attr:`size`.
 
 Args:
-    sizes (int...): a set of integers defining the shape of the output tensor
+    size (int...): a tuple of integers defining the shape of the output tensor.
     out (Tensor, optional): the output tensor
+    dtype (:class:`torch.dtype`, optional): the desired type of returned tensor.
+        Default: if None, uses a global default (see :func:`torch.set_default_tensor_type`)
+    layout (:class:`torch.layout`, optional): the desired layout of returned Tensor.
+        Default: :class:`torch.strided`.
+    device (:class:`torch.device`, optional): the desired device of returned tensor.
+        Default: if None, uses a global default (see :func:`torch.set_default_tensor_type`)
```

Comment:
there's no global default device.  For example, different values of `with.cuda.device_of` will yield a tensor on different gpus.  This should say something like uses the current device for the corresponding default tensor type (i.e. this will be CPU for CPU tensor types, the current cuda device for cuda tensor types).